apps/core/lib/core/schema/user.ex

```diff
@@ -161,8 +161,13 @@ defmodule Core.Schema.User do
   end
 
   def set_email_changed(cs, model) do
+    confirm_by = Timex.now() |> Timex.shift(days: 7)
+
     case get_change(cs, :email) do
-      email when (not is_nil(email)) and (email != model.email) -> put_change(cs, :email_changed, true)
+      email when (not is_nil(email)) and (email != model.email) ->
```

Comment:
oh i didn't notice this but the email != email will be covered by get_change (it shouldn't return a value if there's no change I believe).

Also it's usually better to match `email when is_binary(email)` since a positive match is more robust

Reply:
Okay, I wasn't sure about it and that's why it's here. I'll update.